systemtests/src/test/java/io/enmasse/systemtest/isolated/bridging/ForwardersTest.java

```diff
@@ -0,0 +1,132 @@
+package io.enmasse.systemtest.isolated.bridging;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.Test;
+import io.enmasse.address.model.Address;
+import io.enmasse.address.model.AddressBuilder;
+import io.enmasse.address.model.AddressSpace;
+import io.enmasse.address.model.AddressSpecForwarderBuilder;
+import io.enmasse.address.model.AddressSpecForwarderDirection;
+import io.enmasse.systemtest.UserCredentials;
+import io.enmasse.systemtest.amqp.AmqpClient;
+import io.enmasse.systemtest.bases.bridging.BridgingBase;
+import io.enmasse.systemtest.model.address.AddressType;
+import io.enmasse.systemtest.model.addressplan.DestinationPlan;
+import io.enmasse.systemtest.time.TimeoutBudget;
+import io.enmasse.systemtest.utils.AddressUtils;
+import io.enmasse.systemtest.utils.TestUtils;
+
+public class ForwardersTest extends BridgingBase {
+
+    //tested usecases
+    //Forwarding messages from a local queue in a local address space to a destination on a remote AMQP endpoint
+    //Forwarding messages to a local queue in a local address space from a destination on a remote AMQP endpoint
+
+    //forward to FULL remote queue
+    //forward from FULL remote queue
+
+    private static final String REMOTE_QUEUE1 = "queue1";
+    private static final String REMOTE_QUEUE2 = "queue2";
+
+    @Override
+    protected String[] remoteBrokerQueues() {
+        return new String[] {REMOTE_QUEUE1, REMOTE_QUEUE2};
+    }
+
+    @Test
+    public void testForwardToRemoteQueue() throws Exception {
+        AddressSpace space = createAddressSpace("forward-to-remote", "*");
+        Address forwarder = new AddressBuilder()
+                .withNewMetadata()
+                .withName(AddressUtils.generateAddressMetadataName(space, "forwarder-queue1"))
+                .withNamespace(kubernetes.getInfraNamespace())
+                .endMetadata()
+                .withNewSpec()
+                .withAddress("forwarder-queue1")
+                .withType(AddressType.QUEUE.toString())
+                .withPlan(DestinationPlan.STANDARD_SMALL_QUEUE)
+                .addToForwarders(new AddressSpecForwarderBuilder()
+                        .withName("forwarder1")
+                        .withRemoteAddress(REMOTE_NAME + "/" + REMOTE_QUEUE1)
+                        .withDirection(AddressSpecForwarderDirection.out)
+                        .build())
+                .endSpec()
+                .build();
+        resourcesManager.setAddresses(forwarder);
+        AddressUtils.waitForForwardersReady(new TimeoutBudget(1, TimeUnit.MINUTES), forwarder);
+
+        UserCredentials localUser = new UserCredentials("test", "test");
+        resourcesManager.createOrUpdateUser(space, localUser);
+
+        int messagesBatch = 50;
+
+        //send to address with forwarder
+
+        AmqpClient localClient = getAmqpClientFactory().createQueueClient(space);
```

Comment:
I'm not sure this client will get cleaned up properly (applies to connector tests as well I think?)

Reply:
it's cleaned properly, the amqpClientFactory used is managed and closed properly